R/dm.R

```diff
@@ -98,7 +98,8 @@ new_dm <- function(tables, data_model) {
     as.character(data_model_tables$display),
     keys,
     references,
-    filter = new_filters()
+    filter = new_filters(),
+    zoom = new_zoom()
```

Comment:
Can you please get rid of `new_dm2()` as part of this pull request? I remember it's called only from here.

Reply:
Please inline so that `new_dm()` calls `new_dm3()` directly.